src/extension.ts

```diff
@@ -316,13 +316,13 @@ export class Extension implements RunHooks {
       await this._models.createModel(workspaceFolderPath, uriToPath(configFileUri), playwrightInfo);
     }
 
-    await this._models.ensureHasEnabledModels();
+    this._models.ensureHasEnabledModels();
     this._testTree.finishedLoading();
     return configFiles;
   }
 
   private _modelsUpdated() {
-    this._updateVisibleEditorItems();
+    void this._updateVisibleEditorItems();
```

Comment:
This one seems strange - calling async method before calling a few sync ones is usually a mistake. Perhaps we can change the order of the calls? Or make this whole method async?